static/scss/top-app-bar.scss

```
@@ -59,52 +65,163 @@
       }
 
       &.button {
-        background: $primary;
+        @include media-breakpoint-up(md) {
+          background: $primary;
+          color: white;
+          border-radius: 3px;
+          box-shadow: inset 0 0 2px 0 rgba(255, 255, 255, 0.57);
+
+          &:hover {
+            background: black;
+          }
+        }
+      }
+    }
+  }
+
+  @include media-breakpoint-down(sm) {
+    display: none;
+    position: absolute;
+    padding-top: 70px;
+    background: rgba(0, 0, 0, 0.91);
+    left: 0;
+    right: 0;
+    bottom: 0;
+    top: 70px;
+    height: auto !important;
+    z-index: 999;
+
+    li {
+      display: block;
+      padding: 10px 0;
+      margin: 0 auto;
+      max-width: 170px;
+      text-align: left;
+
+      a {
         color: white;
+        font-size: 18px;
+        font-weight: 600;
+        line-height: 25px;
+        padding: 0;
+        margin-top: 5px;
 
         &:hover {
-          background: black;
+          color: white;
         }
       }
     }
   }
 }
 
+.authenticated-menu {
+  display: none !important;
+
+  @include media-breakpoint-down(sm) {
+    display: block !important;
+  }
+}
+
 .navbar-collapse.collapsing,
 .navbar-collapse.show {
   display: flex;
+  @include media-breakpoint-down(sm) {
+    display: block;
+  }
 }
 
 .site-logo {
-  width: 154px;
-  height: 47.5px;
+  width: 65px;
+  height: 35px;
+  @include media-breakpoint-down(sm) {
+    width: 46px;
+    height: 25px;
+  }
 }
 
-.navbar-toggler.nav-opener {
-  border: 1px solid $primary;
-  border-radius: 2px;
-  color: $primary;
-  margin: 5px 0 0;
-  padding: 8px 6px 6px;
+.navbar-brand {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  padding-top: 0;
+
+  .mitx-online-link {
+    font-size: 24px;
+    color: #000000;
+    font-weight: bold;
+    line-height: 24px;
+    letter-spacing: 0;
+    padding-left: 15px;
+
+    @include media-breakpoint-down(sm) {
+      font-size: 18px;
+      line-height: 18px;
+      padding-left: 8px;
+    }
+  }
 
-  .navbar-toggler-icon {
-    width: 18px;
-    height: 15px;
-    display: inline-block;
-    border: 1px solid $primary;
-    border-width: 2px 0;
-    position: relative;
-    margin: -2px 5px 0 0;
-    vertical-align: top;
+  .divider-large {
+    width: 1px;
+    height: 48px;
+    margin: 0 0 0 15px;
+    background: black;
+    @include media-breakpoint-down(sm) {
+      height: 34px;
+      margin: 0 0 0 8px;
+    }
+  }
+}
+
+.navbar-toggler {
+  position: relative;
+  width: 24px;
+  height: 24px;
+
+  ul {
+    cursor: pointer;
+  }
 
-    &:after {
-      position: absolute;
-      left: 0;
-      right: 0;
-      content: "";
+  .bar {
```

Are these `.bar`s responsible for the hamburger menu icon? If they are, any reason we wouldn't just use an icon from material icons instead? https://fonts.google.com/icons?selected=Material+Icons&icon.query=hamburger

---

Yeah, that's responsible. As we are converting that bar to a close icon when the menu opens. So that's compulsory to follow it.

---

Comment here would help
```suggestion
  // This series of .bar elements creates a hamburger menu that tranforms into an "X" when clicked
  .bar {
```